project-base/storefront/pages/index.tsx

```diff
@@ -42,8 +52,20 @@ const HomePage: NextPage<ServerSidePropsType> = () => {
                 </Webline>
 
                 <Webline className="mb-6">
-                    <h2 className="mb-3">{t('Promoted products')}</h2>
-                    <PromotedProducts />
+                    {isLuigisBoxActive ? (
+                        <>
+                            <h2 className="mb-3">{t('Recommended for you')}</h2>
+                            <RecommendedProducts
+                                itemUuids={[]}
```

Comment:
Hmm, why not to set this empty array as default value in RecommendedProducts component? And don't send this prop if you don't have any data.